src/main/java/com/alvarium/annotators/vulnerability/OsvApiHandler.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright 2023 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package com.alvarium.annotators.vulnerability;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+
+import com.google.gson.Gson;
+
+public class OsvApiHandler implements VulnerabilityApiHandler {
+    final private Gson gson = new Gson();
+    final private String endpoint;
+
+    public OsvApiHandler() {
+        this.endpoint = "https://api.osv.dev/v1/querybatch";

```

Comment:
This should probably be placed in the SDK config file. We don't currently have Annotator-specific config entries, so this is an interesting problem.

Reply:
Yes, I imagine that we'd also need to specify which vulnerability provider we use in the config. Would you like me to look into designing and implementing support for annotation-specific configuration before continuing the review for this PR ?

Reply:
Opened #110 to address this problem.